app/javascript/components/event/InviteInfo.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import Axios from 'axios';
+import Grid from '@material-ui/core/Grid';
+import { FormButton } from '../../ui/Buttons';
+const queryString = require('query-string');
+
+export default class InviteInfo extends React.Component {
+  constructor(props) {
+    super(props);
+    const parsed = queryString.parse(this.props.location.search);
+    this.state = { invite: { id: '', event: {}, token: parsed.token }};
+    this.fetchAvailableEvent = this.fetchAvailableEvent.bind(this);
+    this.handleAccess = this.handleAccess.bind(this);
+    this.handleReject = this.handleReject.bind(this);
+  }
+
+  fetchAvailableEvent() {
+    let headers = {};
+    if (localStorage.user) {
+      headers = JSON.parse(localStorage.user);
+    }
+    const params = this.props.match.params;
+    const inviteShowUrl = `/api/v1/events/${params.event_id}/event_invites/${params.id}`;
+    const inviteUrl = `events/${params.event_id}/event_invites/${params.id}`;
+    Axios.get(inviteShowUrl, { params: { token: this.state.invite.token }, headers: headers })
+      .then(response => { this.setState({ invite: response.data });})
+      .catch(error => {
+        if (error.response.statusText === 'Unauthorized')
+          this.props.history.push(`/sign_in?redirect_url=${inviteUrl}&token=${this.state.invite.token}`);
+      });
+  }
+
+  componentDidMount() {
+    this.fetchAvailableEvent();
+  }
+
+  handleAccess = () => {
+    let headers = {};
+    if (localStorage.user) {
+      headers = JSON.parse(localStorage.user);
+    }
+    headers['X-CSRF-Token'] = document.querySelector('meta[name=\'csrf-token\']').getAttribute('content');
+    const params = this.props.match.params;
+    const inviteRejectUrl = `/api/v1/events/${params.event_id}/event_invites/${params.id}`;
+    const parsed = queryString.parse(this.props.location.search);
+    Axios.patch(inviteRejectUrl, { status: 'access', token: parsed.token }, { headers: headers })
+      .then(() => {
+        Axios.post(`/api/v1/events/${this.props.match.params.event_id}/attendances`, { }, { headers: headers })
```

Comment:
why data is empty?

Reply:
for creating attendance i need only event_id from url and user_id from current_user, so i don't know what i can do with data in request